### DISCUSSION TITLE
Slow Downloading issue

### DISCUSSION THREAD

**Discussion by Clarkwayne1** [aniyomiorg/aniyomi#1224] [General]
If I am watching anime from any extension the vedio is going smoothly but if I download vedio it downloading speed is very slow so what I do help me

**Comment by Dark-Knight987** [comment]
Same with gogoanime/anitaku

**Comment by searingsquid** [comment]
It's the same with yomiroll

**Reply by LittleFreak** [reply]
Yomiroll has fairly the highest quality /biggest files, so it's forgiven, isn't it? 😉
